helidon-standalone-quickstart-mp does not support GraalVM native-image

## Environment Details
* Helidon Version: 2.0.0-RC1
* Helidon MP
* JDK version: GraalVM Version 20.1.0 (Java Version 11.0.7)
* OS: Ubuntu 16.04 LTS
* Docker version (if applicable):

----------

## Problem Description
profile for native image is not applicable for helidon-standalone-quickstart-mp. Native image generation fails if it is used. If it is not required to support GraalVM native image for this quick start, its description shall be removed from README.md in the same module. By default profile is not set, so the only thing which does not corresponds is README.md. But otherwise native-image shall be fixed. 

## Steps to reproduce
mvn package -Pnative-image

stack trace:
```
[INFO] --- helidon-maven-plugin:1.1.1:native-image (default) @ helidon-standalone-quickstart-mp ---
[INFO] Building native image :/home/mavines/git/helidon/examples/quickstarts/helidon-standalone-quickstart-mp/target/helidon-standalone-quickstart-mp
[INFO] [helidon-standalone-quickstart-mp:13083]    classlist:  18,297.09 ms,  0.94 GB
[INFO] [helidon-standalone-quickstart-mp:13083]        (cap):   4,673.44 ms,  0.94 GB
[INFO] [helidon-standalone-quickstart-mp:13083]        setup:  12,190.19 ms,  0.94 GB
[INFO] 2020.06.11 14:10:56 INFO io.helidon.microprofile.cdi.LogConfig Thread[ForkJoinPool-2-worker-5,5,main]: Logging at initialization configured using classpath: /logging.properties
[INFO] 2020.06.11 14:10:56 INFO org.jboss.weld.Version Thread[ForkJoinPool-2-worker-5,5,main]: WELD-000900: 3.1.3 (Final)
[INFO] 2020.06.11 14:10:59 INFO org.jboss.weld.Bootstrap Thread[ForkJoinPool-2-worker-5,5,main]: WELD-ENV-000020: Using jandex for bean discovery
[INFO] 2020.06.11 14:10:59 INFO org.jboss.weld.Bootstrap Thread[ForkJoinPool-2-worker-5,5,main]: WELD-000101: Transactional services not available. Injection of @Inject UserTransaction not available. Transactional observers will be invoked synchronously.
[INFO] 2020.06.11 14:10:59 INFO org.jboss.weld.Event Thread[ForkJoinPool-2-worker-5,5,main]: WELD-000411: Observer method [BackedAnnotatedMethod] private io.helidon.microprofile.openapi.OpenApiCdiExtension.processAnnotatedType(@Observes ProcessAnnotatedType<X>) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
[INFO] 2020.06.11 14:10:59 INFO org.jboss.weld.Event Thread[ForkJoinPool-2-worker-5,5,main]: WELD-000411: Observer method [BackedAnnotatedMethod] public org.glassfish.jersey.ext.cdi1x.internal.ProcessAllAnnotatedTypes.processAnnotatedType(@Observes ProcessAnnotatedType<?>, BeanManager) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
[INFO] 2020.06.11 14:12:37 WARNING com.netflix.config.sources.URLConfigurationSource Thread[ForkJoinPool-2-worker-5,5,main]: No URLs will be polled as dynamic configuration sources.
[INFO] 2020.06.11 14:12:37 INFO com.netflix.config.sources.URLConfigurationSource Thread[ForkJoinPool-2-worker-5,5,main]: To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
[INFO] Warning: RecomputeFieldValue.ArrayIndexScale automatic substitution failed. The automatic substitution registration was attempted because a call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) was detected in the static initializer of rx.internal.util.unsafe.ConcurrentCircularArrayQueue. Detailed failure reason(s): Could not determine the field where the value produced by the call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) for the array index scale computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
[INFO] Warning: RecomputeFieldValue.ArrayBaseOffset automatic substitution failed. The automatic substitution registration was attempted because a call to jdk.internal.misc.Unsafe.arrayBaseOffset(Class) was detected in the static initializer of rx.internal.util.unsafe.ConcurrentCircularArrayQueue. Detailed failure reason(s): Could not determine the field where the value produced by the call to jdk.internal.misc.Unsafe.arrayBaseOffset(Class) for the array base offset computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
[INFO] Warning: RecomputeFieldValue.ArrayIndexScale automatic substitution failed. The automatic substitution registration was attempted because a call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) was detected in the static initializer of rx.internal.util.unsafe.SpscUnboundedArrayQueue. Detailed failure reason(s): Could not determine the field where the value produced by the call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) for the array index scale computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
[INFO] Warning: RecomputeFieldValue.ArrayIndexScale automatic substitution failed. The automatic substitution registration was attempted because a call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) was detected in the static initializer of rx.internal.util.unsafe.ConcurrentSequencedCircularArrayQueue. Detailed failure reason(s): Could not determine the field where the value produced by the call to jdk.internal.misc.Unsafe.arrayIndexScale(Class) for the array index scale computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
[INFO] Warning: RecomputeFieldValue.ArrayBaseOffset automatic substitution failed. The automatic substitution registration was attempted because a call to jdk.internal.misc.Unsafe.arrayBaseOffset(Class) was detected in the static initializer of rx.internal.util.unsafe.ConcurrentSequencedCircularArrayQueue. Detailed failure reason(s): Could not determine the field where the value produced by the call to jdk.internal.misc.Unsafe.arrayBaseOffset(Class) for the array base offset computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
[INFO] WARNING GR-10238: VarHandle for static field is currently not fully supported. Static field private static volatile java.lang.System$Logger jdk.internal.event.EventHelper.securityLogger is not properly marked for Unsafe access!
[INFO] [helidon-standalone-quickstart-mp:13083]     (clinit):   1,733.70 ms,  5.17 GB
[INFO] [helidon-standalone-quickstart-mp:13083]   (typeflow): 287,044.42 ms,  5.17 GB
[INFO] [helidon-standalone-quickstart-mp:13083]    (objects): 109,558.26 ms,  5.17 GB
[INFO] [helidon-standalone-quickstart-mp:13083]   (features):   5,873.99 ms,  5.17 GB
[INFO] [helidon-standalone-quickstart-mp:13083]     analysis: 411,785.34 ms,  5.17 GB
[WARNING] Error: Unsupported features in 2 methods
[WARNING] Detailed message:
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor$Worker.thread of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor$Worker@540ef7af reached by 
[WARNING] 	reading field java.util.HashMap$Node.key of
[WARNING] 		constant java.util.HashMap$Node@ed3977f reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@233e4dd9 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@6b8d0779 reached by 
[WARNING] 	reading field java.util.HashSet.map of
[WARNING] 		constant java.util.HashSet@c15667a9 reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workers of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$Node.thread of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$Node@58dde31b reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.lastWaiter of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7aee5a22 reached by 
[WARNING] 	reading field java.util.concurrent.LinkedBlockingQueue.notEmpty of
[WARNING] 		constant java.util.concurrent.LinkedBlockingQueue@41a502e reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workQueue of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] 
[WARNING] com.oracle.svm.core.util.UserError$UserException: Unsupported features in 2 methods
[WARNING] Detailed message:
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor$Worker.thread of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor$Worker@540ef7af reached by 
[WARNING] 	reading field java.util.HashMap$Node.key of
[WARNING] 		constant java.util.HashMap$Node@ed3977f reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@233e4dd9 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@6b8d0779 reached by 
[WARNING] 	reading field java.util.HashSet.map of
[WARNING] 		constant java.util.HashSet@c15667a9 reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workers of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$Node.thread of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$Node@58dde31b reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.lastWaiter of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7aee5a22 reached by 
[WARNING] 	reading field java.util.concurrent.LinkedBlockingQueue.notEmpty of
[WARNING] 		constant java.util.concurrent.LinkedBlockingQueue@41a502e reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workQueue of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] 
[WARNING] 	at com.oracle.svm.core.util.UserError.abort(UserError.java:79)
[WARNING] 	at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:217)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:753)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:538)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$run$0(NativeImageGenerator.java:451)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
[WARNING] Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Unsupported features in 2 methods
[WARNING] Detailed message:
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor$Worker.thread of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor$Worker@540ef7af reached by 
[WARNING] 	reading field java.util.HashMap$Node.key of
[WARNING] 		constant java.util.HashMap$Node@ed3977f reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@233e4dd9 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@6b8d0779 reached by 
[WARNING] 	reading field java.util.HashSet.map of
[WARNING] 		constant java.util.HashSet@c15667a9 reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workers of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] Error: Detected a started Thread in the image heap. Threads running in the image generator are no longer running at image run time.  To see how this object got instantiated use -H:+TraceClassInitialization. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image run time by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Trace: Object was reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$Node.thread of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$Node@58dde31b reached by 
[WARNING] 	reading field java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.lastWaiter of
[WARNING] 		constant java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7aee5a22 reached by 
[WARNING] 	reading field java.util.concurrent.LinkedBlockingQueue.notEmpty of
[WARNING] 		constant java.util.concurrent.LinkedBlockingQueue@41a502e reached by 
[WARNING] 	reading field java.util.concurrent.ThreadPoolExecutor.workQueue of
[WARNING] 		constant java.util.concurrent.ThreadPoolExecutor@443e804b reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader.executor of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEventPreloader@6eab666 reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.events.ContainerLifecycleEvents.preloader of
[WARNING] 		constant org.jboss.weld.bootstrap.events.ContainerLifecycleEvents@1b99c891 reached by 
[WARNING] 	reading field java.util.HashMap$Node.value of
[WARNING] 		constant java.util.HashMap$Node@438e8a63 reached by 
[WARNING] 	indexing into array
[WARNING] 		constant java.util.HashMap$Node[]@c4260b6 reached by 
[WARNING] 	reading field java.util.HashMap.table of
[WARNING] 		constant java.util.HashMap@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry.services of
[WARNING] 		constant org.jboss.weld.bootstrap.api.helpers.SimpleServiceRegistry@1abdeacc reached by 
[WARNING] 	reading field org.jboss.weld.environment.deployment.AbstractWeldDeployment.serviceRegistry of
[WARNING] 		constant org.jboss.weld.environment.deployment.WeldDeployment@7d5cc359 reached by 
[WARNING] 	reading field io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi.deployment of
[WARNING] 		constant io.helidon.microprofile.cdi.HelidonContainerImpl$HelidonCdi@15c47aae reached by 
[WARNING] 	reading field java.util.concurrent.atomic.AtomicReference.value of
[WARNING] 		constant java.util.concurrent.atomic.AtomicReference@7f5f1b39 reached by 
[WARNING] 	scanning method io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] Call path from entry point to io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(): 
[WARNING] 	at io.helidon.microprofile.cdi.HelidonCdiProvider.getCDI(HelidonCdiProvider.java:29)
[WARNING] 	at javax.enterprise.inject.spi.CDI.current(CDI.java:64)
[WARNING] 	at com.oracle.svm.reflect.CDI_current_4f3ca7f3b2c81296f8ef411f769b23ecb5f90faf_1839.invoke(Unknown Source)
[WARNING] 	at java.lang.reflect.Method.invoke(Method.java:566)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:345)
[WARNING] 	at javax.xml.transform.TransformerException.printStackTrace(TransformerException.java:285)
[WARNING] 	at com.oracle.svm.jni.functions.JNIFunctions.ExceptionDescribe(JNIFunctions.java:761)
[WARNING] 	at com.oracle.svm.core.code.IsolateEnterStub.JNIFunctions_ExceptionDescribe_b5412f7570bccae90b000bc37855f00408b2ad73(generated:0)
[WARNING] 
[WARNING] 	at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:129)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:750)
[WARNING] 	... 8 more
[WARNING] Error: Image build request failed with exit status 1
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  08:05 min
[INFO] Finished at: 2020-06-11T14:17:47+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.helidon.build-tools:helidon-maven-plugin:1.1.1:native-image (default) on project helidon-standalone-quickstart-mp: Image generation failed, exit code: 1 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/dis
```

--- a/applications/pom.xml
+++ b/applications/pom.xml
@@ -47,7 +47,7 @@
         <version.plugin.dependency>3.1.2</version.plugin.dependency>
         <version.plugin.exec>1.6.0</version.plugin.exec>
         <version.plugin.failsafe>3.0.0-M5</version.plugin.failsafe>
-        <version.plugin.helidon>2.1.0</version.plugin.helidon>
+        <version.plugin.helidon>2.1.1</version.plugin.helidon>
         <version.plugin.helidon-cli>2.1.0</version.plugin.helidon-cli>
         <version.plugin.jar>3.0.2</version.plugin.jar>
         <version.plugin.os>1.5.0.Final</version.plugin.os>

--- a/examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml
+++ b/examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml
@@ -41,7 +41,7 @@
         <version.plugin.eclipselink>2.7.5.1</version.plugin.eclipselink>
         <version.plugin.exec>1.6.0</version.plugin.exec>
         <version.plugin.failsafe>3.0.0-M5</version.plugin.failsafe>
-        <version.plugin.helidon>2.1.0</version.plugin.helidon>
+        <version.plugin.helidon>2.1.1</version.plugin.helidon>
         <version.plugin.helidon-cli>2.1.0</version.plugin.helidon-cli>
         <version.plugin.jandex>1.0.6</version.plugin.jandex>
         <version.plugin.jar>3.0.2</version.plugin.jar>

--- a/examples/quickstarts/helidon-standalone-quickstart-se/pom.xml
+++ b/examples/quickstarts/helidon-standalone-quickstart-se/pom.xml
@@ -40,7 +40,7 @@
         <version.plugin.dependency>3.0.0</version.plugin.dependency>
         <version.plugin.exec>1.6.0</version.plugin.exec>
         <version.plugin.failsafe>3.0.0-M5</version.plugin.failsafe>
-        <version.plugin.helidon>2.1.0</version.plugin.helidon>
+        <version.plugin.helidon>2.1.1</version.plugin.helidon>
         <version.plugin.helidon-cli>2.1.0</version.plugin.helidon-cli>
         <version.plugin.jar>3.0.2</version.plugin.jar>
         <version.plugin.os>1.5.0.Final</version.plugin.os>

